akka-http-core/src/main/scala/akka/http/model/japi/Http.scala

```diff
@@ -0,0 +1,167 @@
+/*
+ * Copyright (C) 2009-2015 Typesafe Inc. <http://www.typesafe.com>
+ */
+
+package akka.http.model.japi
```

Comment:
So, this just reinstantiates the previous status-quo. But otherwise you are of course right, that the `Http` class should not live in the model.